src/librbd/cache/pwl/AbstractWriteLog.h

```diff
@@ -271,19 +271,23 @@ class AbstractWriteLog {
 
   void pwl_init(Context *on_finish, pwl::DeferredContexts &later);
   void update_image_cache_state(Context *on_finish);
-  void load_existing_entries(pwl::DeferredContexts &later);
   void wake_up();
-  void process_work();
 
+  void update_entries(std::shared_ptr<pwl::GenericLogEntry> log_entry,
```

Comment:
Nit: same comment here and below re: aligning the params. Either pull them all down to the line after the method name or align all params on subsequent lines by the indentation of the first parameter.

Reply:
Done